devel/gnome-builder/Makefile

```diff
@@ -43,6 +43,8 @@ MESON_ARGS=	-Dwith_flatpak=false \
 		-Dwith_qemu=false \
 		-Dwith_jhbuild=false
 
+CFLAGS+=       -Wno-error=return-type
```

Comment:
Is that a new change or a change from upstream that I did not apply?

Reply:
I don't know why I need this to build, but without build failed.

Reply:
resolved by https://github.com/freebsd/freebsd-ports-gnome/blob/master/devel/gnome-builder/Makefile